tracer_test.go

```diff
@@ -174,45 +174,46 @@ var _ = Describe("SpanRecorder", func() {
 		})
 	})
 
-	Context("When tracer has a SpanRecorder", func() {
-		var fakeRecorder *lightstepfakes.FakeSpanRecorder
+	Describe("SpanRecorder", func() {
+		Context("when tracer has a SpanRecorder", func() {
+			var fakeRecorder *lightstepfakes.FakeSpanRecorder
 
-		BeforeEach(func() {
-			fakeRecorder = new(lightstepfakes.FakeSpanRecorder)
-			tracer = NewTracer(Options{
-				AccessToken: accessToken,
-				ConnFactory: fakeGrpcConnection(new(cpbfakes.FakeCollectorServiceClient)),
-				Recorder:    fakeRecorder,
+			BeforeEach(func() {
+				fakeRecorder = new(lightstepfakes.FakeSpanRecorder)
+				tracer = NewTracer(Options{
+					AccessToken: accessToken,
+					ConnFactory: fakeGrpcConnection(new(cpbfakes.FakeCollectorServiceClient)),
+					Recorder:    fakeRecorder,
+				})
 			})
-		})
 
-		AfterEach(func() {
-			closeTestTracer(tracer)
-		})
+			AfterEach(func() {
+				closeTestTracer(tracer)
+			})
```

Comment:
I find these sorts of "spacing" changes suspicious, just make sure `go fmt` is run? It could just be weird formatting before and you are actually fixing it now.